rts/motoko-rts/src/text.rs

```diff
@@ -83,11 +83,15 @@ pub unsafe fn text_concat<M: Memory>(mem: &mut M, s1: Value, s2: Value) -> Value
         let blob2 = s2.as_blob();
 
         let r = alloc_text_blob(mem, new_len);
-        let r_payload: *const u8 = r.as_blob().payload_addr();
```

Comment:
I think this should have been a `*mut` from the get-go.

Reply:
Because of the memcpy below? Seems reasonable.

Reply:
Yeah, it goes into the `dest` arg of memcpy.